modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -34,30 +34,30 @@ With this update, you can use the `developerHub.flavor` field to identify whethe
 [source,yaml,subs="+quotes"]
 ----
 developerHub:
-  flavor: &lt;flavor&gt;;
+  flavor: <flavor>
 ----
 
 `flavor`::
 Identify the flavor of Backstage that is running. Default value: `rhdh`
 
 
 [id="feature-rhidp-4419"]
-== Ability to manage PVCs in RHDH Operator
-
-You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring {product-very-short} Operator.
+== Ability to manage Persistent Volume Claim (PVCs) in {product-very-short} Operator
```

Comment:
release notes are generated using JIRA as the single source of truth. Please apply your changes in https://issues.redhat.com/browse/RHIDP-4419 then run this script:

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.4/modules/release-notes/single-source-release-notes.py